lib/ide/ast/impl/src/lib.rs

```diff
@@ -19,11 +19,12 @@ use shapely::*;
 use uuid::Uuid;
 
 
-
+/// A mapping between text position and immutable ID
 #[derive(Clone,Debug,Default,Deserialize,Eq,PartialEq,Serialize)]
 pub struct IdMap(pub Vec<(Span,ID)>);
 
 impl IdMap {
+    /// Assigns Span to given ID
     pub fn insert(&mut self, span:Span, id:ID) {
         self.0.push((span, id));
```

Comment:
lets refactor this .0 to some named field. Its so hard to raed like this. I have really hard time to understandwhat all these .0 whe nreading this code

Reply:
Is `.0` okay, if the field is private?